loader/src/platform/windows/main.cpp

```diff
@@ -138,11 +138,29 @@ void* mainTrampolineAddr;
 #include "gdTimestampMap.hpp"
 unsigned int gdTimestamp = 0;
 
+// In case the game is launched from a different directory through command line
+// this function will set the current working directory to the game's directory
+// to avoid the game crashing due to not being able to find the resources
+static void fixCWD() {
+    WCHAR cwd[MAX_PATH];
+    DWORD size = GetModuleFileNameW(NULL, cwd, sizeof(cwd));
+    if (size == sizeof(cwd)) return;
+    for (int i = size - 1; i >= 0; i--) {
```

Comment:
im not sure how i feel about this part since it will replace wrong if the string is smth like path\GeometryDash.exe which is bad

Reply:
i do understand it feels kinda wrong but this is the simplest solution i could come up with. I mean, i could convert it to `std::string` and then use `std::find`. But i don't want like half the file to be about fixing the CWD. I feel like this is short and concise.